desc/coils.py

```diff
@@ -220,7 +273,9 @@ class CoilSet(Coil, MutableSequence):
 
     _io_attrs_ = Coil._io_attrs_ + ["_coils"]
 
-    def __init__(self, *coils, name=""):
+    def __init__(
+        self, *coils, name=""
+    ):  # FIXME: if a list of of Coils is passed, this fails...
```

Comment:
Could use `flatten_list` here, on one of my branches I added a flag to also have it flatten tuples.

Reply:
really useful, thanks